### PR TITLE
[RI-212] Document new environment variables and their deprecation status.

### PR DESCRIPTION
**Pull request by GnaneshKunal**
image

CC: @slemeur @nishantgeorge

### CODE REVIEW

File: content/ri/installing/configurations.md
Comment (slemeur) [inline content/ri/installing/configurations.md:50]: should mention where it will be on docker, as it is different.

File: content/ri/installing/configurations.md
Comment (slemeur) [inline content/ri/installing/configurations.md:83]: agreed, should be RILOGDIR

**Review by slemeur — CHANGES_REQUESTED** [review]

File: content/ri/installing/configurations.md
Comment (slemeur) [reply content/ri/installing/configurations.md:83]: so we need to introduce new env variable + deprecate this one.

File: content/ri/installing/configurations.md
Comment (nishantgeorge) [reply content/ri/installing/configurations.md:50]: Yes, good point.

How about:
`~/.redisinsight` on desktop, `/db` on Docker.

**Comment by bmansheim** [comment]
@slemeur This is for the next version of RedisInsight, right?

**Comment by nishantgeorge** [comment]
> @slemeur This is for the next version of RedisInsight, right?

Yes, this is for the next version (not yet released). This PR should be merged only after the release. We will update here to let you know when it's okay to merge.

**Comment by nishantgeorge** [comment]
Retargeting this for `1.8.0`. Do not merge without explicit approval.

**Pull request merged**
